### PR TITLE
E_CLASSROOM-256 [Bugfix] Avatar in the Student List and Friend's Score section is not displaying properly

### PR DESCRIPTION
**Pull request by Teraaay**
### Issue Link

https://framgiaph.backlog.com/view/E_CLASSROOM-256

### Definition of Done
- [x] The avatars of the students in the Students List page and Friend's Score section should already display.

### Related PR

BE Link: https://github.com/framgia/sph-classroom-els-be/pull/48

### Commands to Run
N/A
#### Main note
- N/A
#### Pages Affected
- STUDENTS LIST PAGE:  http://localhost:3003/students
- QUIZ RESULT PAGE:      http://localhost:3003/categories/1/quizzes/1/questions
#### User Interface
- [ ] when `viewing the Students List page and the Quiz Result it` should `display the avatars of the students`
### Screenshots
STUDENTS LIST PAGE
> image

QUIZ RESULT PAGE [Friend's Score Section]
> image

### CODE REVIEW

File: src/pages/student/quizzes/QuizResult/index.js [hunk 1]
```diff
@@ -103,8 +104,9 @@ const QuizResult = ({ score, total, quizId, categoryId }) => {
                                     className={style.friendsAvatar}
                                     alt="avatar"
                                     src={
-                                      friendScore.avatar ||
-                                      'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
+                                      friendScore.avatar
+                                        ? friendScore.avatar_url
+                                        : 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
```
Comment (karlolee50) [inline src/pages/student/quizzes/QuizResult/index.js:109]:
```suggestion
                                      friendScore.avatar || 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
```

File: src/pages/student/students/StudentList/index.js [hunk 2]
```diff
@@ -119,9 +119,9 @@ const StudentList = () => {
             <img
               alt="avatar"
               src={
-                student.avatar === null
-                  ? 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
-                  : student.avatar
+                student.avatar
+                  ? student.avatar_url
+                  : 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
```
Comment (karlolee50) [inline src/pages/student/students/StudentList/index.js:124]:
```suggestion
                student.avatar || 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'
```

**Comment by CaneteJoash** [comment]
### Working
- [x] Able to display the avatar of the user in 'student list'
- [x] Able to display the avatar in the 'quiz result page'

### Comment/s
- [x] In figma the avatar must have the radius of 50% so please try to change the radius of the avatar in STUDENT LIST.

Please refer to the screenshot below.
     
### Screenshot/s

- QUIZ RESULT PAGE

image

- STUDENT LIST

image

- FIGMA

image


------
- Total Test Time: 0.25 hr

**Comment by CaneteJoash** [comment]
### Working
- [x] Able to display the avatar of the user in 'student list'
- [x] Able to display the avatar in the 'quiz result page'
- [x] The avatar radius was fix in 'student list'

Please refer to the screenshot below.
     
### Screenshot/s

- QUIZ RESULT PAGE

image

- STUDENT LIST

image


- FIGMA

image


------
- Total Test Time: 0.25 hr
- Total Comment/s : 1
- Total Iterations: 1

**Review by karlolee50 — APPROVED** [review]
Review time: 10 minutes
Comment/s: 0

**Pull request merged**